tests/conftest.py

```diff
@@ -16,6 +16,8 @@
 
 import cript
 
+HAS_INTEGRATION_TESTS_ENABLED: bool = False
```

Comment:
This is great. 
much needed automation of this. 

One small thing. Maybe the default here is True. But we load and env variable that can override it to false. And in the automated tests we set that variable. 

The advantage being, that no code has to be changed to run the full tests locally.

Reply:
yeah I can put that in a new PR and we can always set it to `True` or `False` ourselves to override it and make testing easy if needed